pkg/asset/manifests/network.go

```diff
@@ -7,18 +7,28 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
 
 	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 	"github.com/openshift/installer/pkg/asset/templates/content/openshift"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/aws"
 	"github.com/openshift/installer/pkg/types/powervs"
 )
 
+const (
+	// Cluster Network MTU for AWS Local Zone deployments on edge machine pools
+	ovnKNetworkMtuEdge   = 1200
+	ocpSDNNetworkMtuEdge = 1250
```

Comment:
perhaps you could type the constant to avoid having to convert when the variable is used.  
`ovnKNetworkMtuEdge  uint32 = 1200
ocpSDNNetworkMtuEdge uint32 = 1250`